### ISSUE TITLE
Train AESOP Loss

### ISSUE THREAD

**Issue by HaoZhang-SX** [2minkyulee/AESOP-Auto-Encoded-Supervision-for-Perceptual-Image-Super-Resolution#5]
What is the difference between  the two scripts below？
Why freeze decoder and without LR Loss in 'train_Realworld_AE_RRDB_DecoderFreeze.yml',  unfreeze decoder and use LR Loss in 'train_Synthetic_AE_RRDB_LRrecon1' ?
Do I need to use all the two scripts in sequence? 
Thanks！

PYTHONPATH="./:${PYTHONPATH}" CUDA_VISIBLE_DEVICES=0,1,2,3 python -m torch.distributed.launch --nproc_per_node=4 --master_port=5678 basicsr/train.py -opt options/train/AutoEncoder/train_Synthetic_AE_RRDB_LRrecon1.yml --launcher pytorch
PYTHONPATH="./:${PYTHONPATH}" CUDA_VISIBLE_DEVICES=0,1,2,3 python -m torch.distributed.launch --nproc_per_node=4 --master_port=5678 basicsr/train.py -opt options/train/AutoEncoder/train_Realworld_AE_RRDB_DecoderFreeze.yml --launcher pytorch

**Comment by 2minkyulee** [comment]
#1
Regarding train_Realworld_AE_RRDB_DecoderFreeze.yml,
We did not use LR loss since the current training pipeline uses the input LR image as GT for the LR loss.
That is, under RealESRGAN-like training pipeline, the GT for LR loss becomes noisy. 
Thus we simply trained the AE in an end-to-end manner while freezing the Decoder (RealESRNet).


#2 
You don't need both of them.
For real-world SR tasks, use train_Realworld_....  (note that it uses a pretrained **Real**ESRNet)
For synthetic (bicubic) SR tasks, use train_Synthetic_...
